static/src/js/routes/Preferences/Preferences.jsx

```diff
@@ -20,7 +59,10 @@ export const Preferences = () => {
       <div className="route-wrapper route-wrapper--light route-wrapper--content-page">
         <div className="route-wrapper__content">
           <div className="route-wrapper__content-inner">
-            <PreferencesContainer />
+            <PreferencesComponent
```

Comment:
This PreferencesComponent is like the old container, the only thing it is doing is lazy loading the form, and passing the props down. I think we can delete that component and load the PreferencesForm here directly, load the zustand values within the form component and avoid passing any props.

We don't need to lazy load the PreferencesForm component, this route component is being lazy loaded in App.jsx